packages/vscode-docker-registries/src/clients/GenericRegistryV2/GenericRegistryV2DataProvider.ts

```diff
@@ -57,19 +83,58 @@ export class GenericRegistryV2DataProvider extends RegistryV2DataProvider {
         return this.authenticationProviders.get(registry)!;
     }
 
-    private addTrackedRegistry(): void {
-        //const trackedRegistryStrings = this.extensionContext.globalState.get<string[]>(TrackedRegistriesKey, []);
+    private async addTrackedRegistry(): Promise<void> {
+        const wizardContext: GenericRegistryV2WizardContext = {
+            registryPrompt: vscode.l10n.t('Registry URL'), // TODO: change prompt
+            usernamePrompt: vscode.l10n.t('Registry Username'), // TODO: change prompt
+            secretPrompt: vscode.l10n.t('Registry Personal Access Token'), // TODO: change prompt

```

Comment:
I'd use "Registry password or personal access token" here since it can be either